tests/10-vcdm2.js

```diff
@@ -443,12 +496,40 @@ describe('Verifiable Credentials Data Model v2.0', function() {
         // eslint-disable-next-line max-len, no-unused-vars
         const regexp = /-?([1-9][0-9]{3,}|0[0-9]{3})-(0[1-9]|1[0-2])-(0[1-9]|[12][0-9]|3[01])T(([01][0-9]|2[0-3]):[0-5][0-9]:[0-5][0-9](\.[0-9]+)?|(24:00:00(\.0+)?))(Z|(\+|-)((0[0-9]|1[0-3]):[0-5][0-9]|14:00))/;
       });
+    });
+  }
+});
+
+// 4.9 Securing Mechanisms https://w3c.github.io/vc-data-model/#securing-mechanisms
+describe('Securing Mechanisms', function() {
+  setupMatrix.call(this);
+  for(const [name, implementation] of match) {
+    const endpoints = new TestEndpoints({implementation, tag});
+    const createOptions = {challenge};
```

Comment:
looks like something changed and the challenge is no longer being used here?

Reply:
Fixed in 5d6be7b. Sorry I missed those!